packages/atomic/src/components/insight/atomic-insight-result-action/atomic-insight-result-action.tsx

```diff
@@ -1,11 +1,30 @@
-import {Component, Prop, State, h} from '@stencil/core';
-import Quickview from '../../../images/quickview.svg';
+import {Result} from '@coveo/headless';
+import {Component, Event, EventEmitter, Prop, State, h} from '@stencil/core';
+import AttachIcon from '../../../images/attach.svg';
+import CopyIcon from '../../../images/copy-dark.svg';
+import EmailIcon from '../../../images/email.svg';
+import QuickviewIcon from '../../../images/preview.svg';
```

Comment:
There's already a `Quickview` icon in the project. Here we add a new one called `preview`, but with a named import for `Quickview`. 

I think we should delete `preview.svg` ?

Reply:
They look extremely similar (exact same?) also when you compare them visually.